tests/ostream_operator.cpp

```diff
@@ -1,7 +1,15 @@
+#include "datamodel/ExampleForCyclicDependency1.h"
 #include "datamodel/ExampleForCyclicDependency1Collection.h"
+#include "datamodel/ExampleForCyclicDependency2.h"
 #include "datamodel/ExampleForCyclicDependency2Collection.h"
```

Comment:
The usual recommendation is to only include the `Collection` header since that brings in all the rest (and you would have to include quite a few headers if you want all the user facing classes otherwise).

Reply:
Yeah good luck telling that to IWYU :smile: 
Fixed